ScnSoft-Education/LeetCodeTraining/SwapNodesInPairs-24/SwapNodesInPairsSolution.cs

```diff
@@ -16,8 +16,7 @@ public class SwapNodesInPairsSolution
     {
         public static ListNode SwapPairs(ListNode head)
         {
-            ListNode result = new ListNode(0);
-            result.next = head;
+            ListNode result = new ListNode(0,head);
```

Comment:
Please format the code.

Reply:
Excuse me, I think you saw previous changes, because string.Empty is already added to the "result" line and the code is already formatted.

Reply:
Yes, you are right, actually string.Empty was applied. Thanks.
But formatting Ctrl + K, F is still not applied for this line.